File: Python/8py 2/get_size.py
"""
|--------------------------------------------------------------------------
| Surface Area and Volume of a Box
|--------------------------------------------------------------------------
|
"""

def get_size(w,h,d):
   return [2 * (w * h + w * d + h * d), w * h * d]


def get_size(w,h,d):
   return [2*d*w + 2*h*w + 2*d*h, w*h*d]


def get_size(w,h,d):
   area = 2 * (w * h + w * d + h * d)
   volume = w * h * d
   return [area, volume]


print(get_size(4, 2, 6), [88,48])
print(get_size(1, 1, 1), [6,1])
print(get_size(1, 2, 1), [10,2])

File: JavaScript/8js 1/getSize.js
/*
|--------------------------------------------------------------------------
| Surface Area and Volume of a Box
|--------------------------------------------------------------------------
|
*/

function getSize(w, h, d) {
   return [2 * (w * h + w * d + h * d), w * h * d]
}


function getSize(w, h, d) {
   return [((w + h) << 1) * d + (w * h << 1), w * h * d];
}


function getSize(w, h, d) {
   return [2 * w * h + 2 * w * d + 2 * d * h, w * h * d]
}


console.log(getSize(4, 2, 6), [88, 48]);
console.log(getSize(10, 10, 10), [600, 1000]);